src/System.Runtime.Intrinsics/src/System/Runtime/Intrinsics/Types.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    // 128 bit types
+    [StructLayout(LayoutKind.Sequential, Size = 16)]
+    public struct Vector128<T> where T : struct 
```

Comment:
Maybe they should be in their own files that is good for extending to future ISA support (i.e. `Vector512<T>`)?